[2022-04-13 22:10:13,668] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest manual__2022-04-13T21:15:36.839665+00:00 [queued]>
[2022-04-13 22:10:13,683] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest manual__2022-04-13T21:15:36.839665+00:00 [queued]>
[2022-04-13 22:10:13,684] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:10:13,684] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-04-13 22:10:13,685] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:10:13,702] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-04-13 21:15:36.839665+00:00
[2022-04-13 22:10:13,706] {standard_task_runner.py:52} INFO - Started process 7743 to run task
[2022-04-13 22:10:13,711] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'manual__2022-04-13T21:15:36.839665+00:00', '--job-id', '1358', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpr2r558e9', '--error-file', '/tmp/tmpffciialp']
[2022-04-13 22:10:13,712] {standard_task_runner.py:77} INFO - Job 1358: Subtask ingest
[2022-04-13 22:10:13,790] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest manual__2022-04-13T21:15:36.839665+00:00 [running]> on host 13db72185839
[2022-04-13 22:10:13,877] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-04-13T21:15:36.839665+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-13T21:15:36.839665+00:00
[2022-04-13 22:10:13,879] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-13 22:10:13,881] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-04-13 22:10:13,891] {subprocess.py:85} INFO - Output:
[2022-04-13 22:10:13,896] {subprocess.py:89} INFO - ***.cfg
[2022-04-13 22:10:13,897] {subprocess.py:89} INFO - dags
[2022-04-13 22:10:13,898] {subprocess.py:89} INFO - logs
[2022-04-13 22:10:13,898] {subprocess.py:89} INFO - output.csv
[2022-04-13 22:10:13,899] {subprocess.py:89} INFO - plugins
[2022-04-13 22:10:13,900] {subprocess.py:89} INFO - requirements.txt
[2022-04-13 22:10:13,900] {subprocess.py:89} INFO - scripts
[2022-04-13 22:10:13,900] {subprocess.py:89} INFO - webserver_config.py
[2022-04-13 22:10:13,901] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-13 22:10:13,931] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220413T211536, start_date=20220413T221013, end_date=20220413T221013
[2022-04-13 22:10:13,963] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-13 22:10:13,999] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
